### PR TITLE
[IMP] website: 404 page improvement

### PR DESCRIPTION
**Pull request by karlosvalverde**
[IMP] website: 404 page improvement

Current 404 page template has been improved, by adjusting the alignment
of the elements and making a better use of their previous Bootstrap
syntax, in order to have a more readable template.

Task-2409545

--

This PR stems from a fork of `master` and targets it, and also closes the PR #62693, which was opened as a fork of `14.0`. 

@Cocographique @JKE-be 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/http_routing/views/http_routing_template.xml [hunk 1]
```diff
@@ -108,21 +108,24 @@
                     <section class="pt128 pb104">
                         <div class="container">
                             <div class="row">
-                                <div class="col-md-6 text-center text-md-right my-md-auto mb-5 mb-md-0">
-                                    <img class="img img-fluid" src="/http_routing/static/src/img/404.svg"/>
+                                <div class="col-lg-4 pb32">
```
Comment (rdeodoo) [inline addons/http_routing/views/http_routing_template.xml:111]: As well spotted by @JKE-be , can you add those back or fix it ?
Now the layout is broken if you add text
Before
image
Now
image

**Comment by JKE-be** [comment]
@robodoo delegate=rdeodoo

Since you create this layout a few month ago, you probably have some interesting feedback ...
Else LGTM

File: addons/http_routing/views/http_routing_template.xml (hunk 1 above)
Comment (karlosvalverde) [reply addons/http_routing/views/http_routing_template.xml:111]: Do you mean adding a `align-self-center` in the column and setting the two columns to `col-lg-6`?

image

What do you think, @Cocographique ?

Comment (rdeodoo) [reply addons/http_routing/views/http_routing_template.xml:111]: @karlosvalverde just the vertical align yes, not the col-lg. overall your layout is cleaner, it is a bit less 'brouillon'

Comment (Cocographique) [reply addons/http_routing/views/http_routing_template.xml:111]: If it's about vertical alignment, you can add `.s_image_text `on the `section` 
and `.align-items-center` on the parent `.row`
[https://github.com/odoo/odoo/blob/4f1c6a1b14e2d969c4ca40b9c0181f93163dda2f/addons/website/views/snippets/snippets.xml#L1044](https://github.com/odoo/odoo/blob/4f1c6a1b14e2d969c4ca40b9c0181f93163dda2f/addons/website/views/snippets/snippets.xml#L1044)

Comment (karlosvalverde) [reply addons/http_routing/views/http_routing_template.xml:111]: Done!

**Review by rdeodoo — APPROVED** [review]
LGTM with the add back of the vertical align classes. Overall the layout is cleaner.
Before:
image

After:
image

@robodoo r+

**Pull request closed without merging**